modules/sso/config/launch/setup/73/added/standalone-openshift.xml

```diff
@@ -23,6 +23,7 @@
         <extension module="org.wildfly.extension.core-management"/>
         <extension module="org.wildfly.extension.elytron"/>
         <extension module="org.wildfly.extension.io"/>
+        <!-- ##TRACING_EXTENSION## -->
```

Comment:
@slaskawi Should this be removed too then?

Reply:
ooh yes! I'm on it!

Reply:
Done :) Thanks!